src/data/graphql/queries.graphql

```diff
@@ -650,4 +650,23 @@ query SubgraphActions($colonyAddress: String!) {
       }
     }
   }
+  events(where: { associatedColony_contains: $colonyAddress, name_in: ["TokensMinted(address,address,uint256)"] }) {
```

Comment:
I think it will work with just `TokensMinted`, but I'm fine with this anyway.

It will have to change anyway when we wire in the rest of the events and actions